### PR TITLE
V1.0.0 merge to master

### PR DESCRIPTION
**Pull request by brendandburns**
for realz this time!

@thockin @eparis @zmerlynn

### CODE REVIEW

File: pkg/version/base.go [hunk 1]
```diff
@@ -35,9 +35,9 @@ package version
 
 var (
 	// TODO: Deprecate gitMajor and gitMinor, use only gitVersion instead.
-	gitMajor     string = "0"              // major version, always numeric
-	gitMinor     string = "21.2+"          // minor version, numeric possibly followed by "+"
-	gitVersion   string = "v0.21.2-dev"    // version from git, output of $(git describe)
+	gitMajor     string = "1"              // major version, always numeric
+	gitMinor     string = "0.0+"           // minor version, numeric possibly followed by "+"
```
Comment (zmerlynn) [inline pkg/version/base.go]: Scary.

**Comment by thockin** [comment]
great big LGTM

**Comment by zmerlynn** [comment]
Tag confirmed to be part of commit. Git mechanics upheld. Do it.

**Comment by rimusz** [comment]
awesome news :+1:

**Comment by kgrvamsi** [comment]
Awesome will give a try!!!!

**Pull request merged**